test/integration/kubernetes_deploy_test.rb

```diff
@@ -917,6 +919,19 @@ def test_cronjobs_can_be_deployed
     cronjobs.assert_cronjob_present("my-cronjob")
   end
 
+  def test_jobs_can_be_successful
+    assert_deploy_success(deploy_fixtures("jobs"))
```

Comment:
Why not deploy the job subset of the hello-cloud fixture set instead of having a second fixture set?